<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QDialog" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1064</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1064</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(20, 20, 20);</string>
  </property>
  <widget class="QFrame" name="Right">
   <property name="geometry">
    <rect>
     <x>595</x>
     <y>10</y>
     <width>461</width>
     <height>616</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>475</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="Left">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>10</y>
     <width>100</width>
     <height>616</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="Home_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>access for interactive map</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Icons/Home.png</normaloff>../Icons/Home.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Search_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>63</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>access for interactive map</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Icons/search.png</normaloff>../Icons/search.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Map_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>63</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>access for interactive map</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Icons/map.png</normaloff>../Icons/map.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Setting_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>access for interactive map</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Icons/settings.png</normaloff>../Icons/settings.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Dashboard_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>63</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>access for interactive map</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Icons/dashboard.png</normaloff>../Icons/dashboard.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="Middle">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>0</y>
     <width>471</width>
     <height>616</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>475</width>
     <height>16777215</height>
    </size>
   </property>
   <widget class="QCheckBox" name="TempUnit_checkBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>260</y>
      <width>101</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator {
	width: 100px;
	height: 50px;
	background-color: rgba(0, 0, 0,0);
}


QCheckBox::indicator::checked{
	image: url(&quot;/Users/User/Documents/SkyMate/Icons/clicked_switch.png&quot;);
}

QCheckBox::indicator::unchecked{
	image: url(&quot;/Users/User/Documents/SkyMate/Icons/unclick_switch.png&quot;);
}

QCheckBox { background-color: rgba(0, 0, 0, 0); }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="matric_checkBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>410</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0,0);</string>
    </property>
    <property name="text">
     <string>matric</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="imperial_checkBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>410</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0,0);</string>
    </property>
    <property name="text">
     <string>imperial</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="FullScreen_checkBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator {
	width: 100px;
	height: 50px;
}

QCheckBox::indicator::checked{
	image: url(&quot;/Users/User/Documents/SkyMate/Icons/clicked_switch.png&quot;);
}

QCheckBox::indicator::unchecked{
	image: url(&quot;/Users/User/Documents/SkyMate/Icons/unclick_switch.png&quot;)
}
QCheckBox { background-color: rgba(0, 0, 0, 0); }</string>
    </property>
    <property name="text">
     <string>FullScreen</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0,0);</string>
    </property>
    <property name="text">
     <string>Celsius</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0,0);</string>
    </property>
    <property name="text">
     <string>Fahrenheit</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>291</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">alternate-background-color: rgb(0, 0, 0);
selection-color: rgb(0, 0, 0);
color: rgb(97, 97, 97);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0,0);</string>
    </property>
    <property name="text">
     <string>Screen Size:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>129</y>
      <width>451</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(53, 53, 53);
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="TimeFormat">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>101</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator {
	width: 100px;
	height: 50px;
	background-color: rgba(0, 0, 0,0);
}


QCheckBox::indicator::checked{
	image: url(&quot;/Users/User/Documents/SkyMate/Icons/clicked_switch.png&quot;);
}

QCheckBox::indicator::unchecked{
	image: url(&quot;/Users/User/Documents/SkyMate/Icons/unclick_switch.png&quot;);
}

QCheckBox { background-color: rgba(0, 0, 0, 0); }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>310</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0,0);</string>
    </property>
    <property name="text">
     <string>12 hours </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0,0);</string>
    </property>
    <property name="text">
     <string>24 hours</string>
    </property>
   </widget>
   <zorder>textEdit</zorder>
   <zorder>matric_checkBox</zorder>
   <zorder>imperial_checkBox</zorder>
   <zorder>FullScreen_checkBox</zorder>
   <zorder>comboBox</zorder>
   <zorder>label_3</zorder>
   <zorder>label_2</zorder>
   <zorder>TempUnit_checkBox</zorder>
   <zorder>label</zorder>
   <zorder>TimeFormat</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
